src/main/java/io/jenkins/plugins/pipeline/restful/api/PipelineRestfulAPI.java

```diff
@@ -18,6 +24,7 @@
 import org.kohsuke.stapler.interceptor.RequirePOST;
 
 import java.io.IOException;
+import org.kohsuke.stapler.verb.GET;
```

Comment:
this is an unused import statement

Reply:
You did't make any changes in this file. So I guess you just just clean all these new imports.

Reply:
Yes resolved it :)